### ISSUE TITLE
Running into issues trying to reproduce the results 😅 (Need help with pretrained models/demo)

### ISSUE THREAD

**Issue by nikou-arch** [ICSResearch/HUNet#1]
Hey guys! 👋

First off, really cool work on HUNet! I've been reading through the paper and the methodology looks super promising.

I'm trying to take this for a spin to see how it performs on my end, but I hit a bit of a wall. I couldn't find the **pre-trained weights** anywhere in the repo. Training from scratch is taking forever on my GPU 🐢... is there any chance you could share a Google Drive or Baidu link for the checkpoints?

Also, if you have a quick `demo.py` or a snippet showing how to format the input data (dimensions, normalization, etc.), that would be a lifesaver. I want to make sure I'm feeding the model the right stuff before debugging further.

Thanks a ton for open-sourcing this! 🚀
